products/ocp4/profiles/stig.profile

```diff
@@ -21,6 +21,8 @@ description: |-
 
 filter_rules: '"ocp4-node" not in platforms and "ocp4-master-node" not in platforms and "ocp4-node-on-sdn" not in platforms and "ocp4-node-on-ovn" not in platforms'
 
+extends: stig-v1r1
+
 selections:
```

Comment:
Remove these selections since they're going to be inherited from the extended profile.